### DISCUSSION TITLE
Launch fancybox programmatically

### DISCUSSION THREAD

**Discussion by get-0ut** [fancyapps/ui#412] [Q&A]
I am trying to learn fancybox, on website I saw this "Launch Fancybox programmatically" but did not get this, kindly share some kindly of docs and example...

new Fancybox([
  {
    src: "<p>Lorem ipsum dolor sit amet.</p>",
    type: "html",
  },
]);

**Comment by fancyapps** [comment]
Hi,

Sorry, but I just do not understand what is so difficult here. Could you please clarify which of the steps you were unclear about? Because all you had to do is to include 1 JS and 1 CSS file, and then copy/paste the code snippet from the homepage -

https://jsfiddle.net/p83ka2zs/
